samples/bluetooth/direct_test_mode/src/dtm.c

```diff
@@ -441,6 +424,30 @@ static uint8_t const dtm_prbs_content[] = {
 	0x8A, 0x84, 0x39, 0xF4, 0x36, 0x0B, 0xF7
 };
 
+static const struct dtm_supp_features supported_features = {
+	.data_len_ext = true,
+	.phy_2m = true,
+	.stable_mod = false,
+#if CONFIG_HAS_HW_NRF_RADIO_BLE_CODED
+	.coded_phy = true,
```

Comment:
It can be even simplified further :) :
```suggestion
	.coded_phy = IS_ENABLED(CONFIG_HAS_HW_NRF_RADIO_BLE_CODED),
```